### ISSUE TITLE
FileNotFoundError

### ISSUE THREAD

**Issue by geek-APTX4869** [yyliu01/TraCoCo#11]
No such file or directory: 'TraCoCo-main/Code/UnetBRATS/Dataloader/data/val.list

**Comment by yyliu01** [comment]
Hi @geek-APTX4869 

Thank you for your interest! I believe there was an mis-operation during upload, and valid.list was accidentally omitted. It should contain the remainder of the data not included in train.list or test.list.

For simplicity, please use test.list directly for the final results, as the reported performance in the paper is based on this test set.

Cheers,
Yuyuan

**Comment by geek-APTX4869** [comment]
But cannot use `test.list` during training
